tools/webtransport/h3/webtransport_h3_server.py

```diff
@@ -35,11 +36,59 @@
 _doc_root: str = ""
 
 
+# TODO(yutakahirano): Use aioquic's H3Capsule when
+# https://github.com/aiortc/aioquic/pull/229 is accepted.
```

Comment:
Is this TODO still available? The PR was closed without merging. It looks like this and the following changes depend on the PR and if PR isn't going to merge, I guess we might use different approach to implement session_closed hander?

Reply:
This is no longer needed as described at https://github.com/aiortc/aioquic/pull/229.

https://github.com/aiortc/aioquic/pull/229 consists of two parts:

1) SETTINGS negotiation changes
2) H3Capsule logic

1) was not needed at all. I thought the new protocol changed the SETTINGS value but I was wrong.
2) We have the same logic in this change. I thought it would be benefitial to have HTTP Datagrams because I'd need to have some HTTP Datagrams logic in aioquic. But my assumption was wrong, we probably won't need to add HTTP Datagrams logic there.

They are why I closed the PR and this PR doesn't depend on it.

Reply:
I see.